packages/components/table/src/table-body/events-helper.ts

```diff
@@ -115,8 +115,17 @@ function useEvents<T>(props: Partial<TableBodyProps<T>>) {
      *    - Expected: 188
      *    - Actual: 188.00000762939453
      */
-    const rangeWidth = Math.round(range.getBoundingClientRect().width)
-    const rangeHeight = Math.round(range.getBoundingClientRect().height)
+    let rangeWidth = range.getBoundingClientRect().width
+    let rangeHeight = range.getBoundingClientRect().height
+    const offsetWidth = rangeWidth - Math.floor(rangeWidth)
+    if (offsetWidth > 0 && offsetWidth < 0.001) {
```

Comment:
`offsetWidth` must be greater than or equal to 0, so there is no need to judge that it is less than 0.
```suggestion
    if (offsetWidth < 0.001) {
```